### PR TITLE
Documentation: Spanish translation of perplexity.mdx

### PR DESCRIPTION
**Pull request by aaronjimv**
# What does this PR do?

Add the Spanish version of perplexity.mdx to transformers/docs/source/es

Fixes #15947 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@omarespejel @sgugger @osanseviero

### CODE REVIEW

File: docs/source/es/perplexity.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+-->
+
+# Perplejidad de los modelos de longitud fija
+
+[[open-in-colab]]
+
+La perplejidad, perplexity en inglés (PPL), es una de las métricas más comunes para evaluar modelos de lenguaje. Antes de sumergirnos, debemos tener en cuenta que esta métrica se aplica específicamente a modelos de lenguaje clásicos (a veces llamados modelos autorregresivos o causales) y no está bien definida para modelos de lenguaje enmascarados como BERT (ver [resumen del modelo](model_summary)).
+
+La perplejidad se define como la media negativa exponenciada de la verosimilitud de una secuencia. Si tenemos una secuencia tokenizada \(X = (x_0, x_1, \dots, x_t)\), entonces la perplejidad de \(X\) es,
```
Comment (stevhliu) [inline docs/source/es/perplexity.md:23]:
```suggestion
La perplejidad se define como la media negativa exponenciada de la verosimilitud de una secuencia. Si tenemos una secuencia tokenizada \\(X = (x_0, x_1, \dots, x_t)\\), entonces la perplejidad de \\(X\\) es,
```

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for your contribution! 🤗

File: docs/source/es/perplexity.md (hunk 1 above)
Comment (aaronjimv) [reply docs/source/es/perplexity.md:23]: Hi thanks! I will fix it.

File: docs/source/es/perplexity.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+-->
+
+# Perplejidad de los modelos de longitud fija
+
+[[open-in-colab]]
+
+La perplejidad, perplexity en inglés (PPL), es una de las métricas más comunes para evaluar modelos de lenguaje. Antes de sumergirnos, debemos tener en cuenta que esta métrica se aplica específicamente a modelos de lenguaje clásicos (a veces llamados modelos autorregresivos o causales) y no está bien definida para modelos de lenguaje enmascarados como BERT (ver [resumen del modelo](model_summary)).
+
+La perplejidad se define como la media negativa exponenciada de la verosimilitud de una secuencia. Si tenemos una secuencia tokenizada \\(X = (x_0, x_1, \dots, x_t)\\), entonces la perplejidad de \\(X\\) es,
```
Comment (osanseviero) [inline docs/source/es/perplexity.md:23]:
```suggestion
La perplejidad se define como la media negativa exponenciada del log-likelihood de una secuencia. Si tenemos una secuencia tokenizada \\(X = (x_0, x_1, \dots, x_t)\\), entonces la perplejidad de \\(X\\) es,
```

Comment (osanseviero) [reply docs/source/es/perplexity.md:23]: `verosimilitud` is a strange translation for log-likelihood I think. Would you mind updating it to log-likelihood to keep same term please?

**Review by osanseviero — APPROVED** [review]
I did a very quick high-level review and it looks good! Thank you :rocket:

File: docs/source/es/perplexity.md (hunk 2 above)
Comment (aaronjimv) [reply docs/source/es/perplexity.md:23]: Hi @osanseviero thanks. Yes, I thought the same thing, I used `verosimilitud` based on [this.](https://www.proz.com/kudoz/english-to-spanish/computers-software/6865844-negative-log-likelihood.html). But I like more to keep  `log-likelihood`.

**Review by stevhliu — APPROVED** [review]
Thanks again! Last step is to run `make style` to fix the failing CI test, and then we can merge :)

**Pull request merged**